cms/blog/templates/cms_blog/blog_index_category_buttons.html

```diff
@@ -0,0 +1,13 @@
+<h4>Categories</h4>
+{% for categ in categories %}
+    {% if categ.name_en == category %}
+        <div class="btn btn-sm btn-topic active">
+            <a href="?category=" role="button">{{ categ }} &nbsp; X</a>
```

Comment:
One could think about hiding the link text from a screen reader here and provide an aria-label here (eg 'remove category {{ categ }}') to make it nicer for screen reader users.